Doc/library/shutil.rst

```diff
@@ -680,6 +682,34 @@ The resulting archive contains:
     -rw-r--r-- tarek/staff   37192 2010-02-06 18:23:10 ./known_hosts
 
 
+.. _shutil-archiving-example-with-basedir:
+
+Archiving example with *base_dir*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, we try to make the use of *base_dir* a bit more clear.  We
```

Comment:
I don't think you need to explain *why* this example is here, just explain *how* to use base_dir.